packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,196 @@
+# Chip Decisions <!-- omit in toc -->
+
+Chips embody a collection of selectable entities that empower users to engage in choices, sift through content, and invoke pertinent actions.
+
+This document outlines the API details of the `Chip` and `ChipGroup` components, encompassing their structural composition, functional attributes, and visual representation.
+
+<img width="100%" src="./chip-variants.png" alt="Different variants of the Chip component">
+
+- [Design](#design)
+- [`Chip` API](#chip-api)
+- [`ChipGroup` API](#chipgroup-api)
+- [Examples](#examples)
+  - [Basic/Uncontrolled Usage](#basicuncontrolled-usage)
+  - [Controlled Usage](#controlled-usage)
+- [Accessibility Considerations](#accessibility-considerations)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=52377%3A27686&mode=design&t=pFzCay0x4a6sTcoQ-1) to all variants of the `Chip` component.
+
+## `Chip` API
+
+> **Note**
+>
+> The `Chip` component is inherently tied to the `ChipGroup` and cannot be utilized outside its context. This is because the chips are designed to be used in multiples since they represent collections of objects.
+
+| Prop                    | Type                                   | Default     | Description                                                                                                                                        | Required |
```

Comment:
Can a chip be dismissible? Additionally, what distinguishes a tag from a chip at its core?

Reply:
@snitin315 can you add the differences from the design doc here?

Reply:
I'll add it in the doc too.

<img width="971" alt="Screenshot 2023-08-28 at 5 41 04 PM" src="https://github.com/razorpay/blade/assets/46647141/a2ac2fdb-dfc3-49ae-bb29-ecee87552f3d">

Reply:
@snitin315 can you add this to tags and badges API doc as well?

Reply:
Will add in separate PR.